services/platform/platform.go

```diff
@@ -144,6 +146,43 @@ func (s *Service) SignIn(ctx context.Context, email, password string) error {
 	return nil
 }
 
+// SignOut logouts that instance form Percona Platform account and removes session id.
```

Comment:
`form` -> `from`

Reply:
Arghhh... that typo follows me.

Fixed.